engine/servermaster/etcd_test.go

```diff
@@ -53,7 +53,11 @@ func TestStartEtcdTimeout(t *testing.T) {
 	cfgCluster.Name = name1
 	cfgCluster.DataDir = dir
 	peer1 := allocTempURL(t)
-	peer2 := allocTempURL(t)
+	peer2 := peer1
```

Comment:
I think the main reason we get this error is `peer1 is not used when allocate peer2`.

Reply:
Need change the pr discription  before merge this pr.